src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsSynchronisationIntTest.kt

```diff
@@ -185,11 +189,11 @@ class NonAssociationsSynchronisationIntTest : SqsIntegrationTestBase() {
 
         @Test
         fun `will not create the non-association in the non-associations service`() {
-          nonAssociationsApi.verify(exactly(0), postRequestedFor(urlPathEqualTo("/sync")))
+          nonAssociationsApi.verify(exactly(0), putRequestedFor(urlPathEqualTo("/sync")))
         }
 
         @Test
-        fun `will not create telemetry tracking`() {
+        fun `will not create telemetry tracking success`() {
           verify(telemetryClient, Times(0)).trackEvent(
             eq("non-association-created-synchronisation-success"),
```

Comment:
the customevent name has changed to non-association-upserted-synchronisation-success

Reply:
ha ha yes - I thought I'd done a full find and replace - will fix in next PR